sopel/modules/weather.py

```diff
@@ -178,6 +178,7 @@ def update_woeid(bot, trigger):
         city = first_result.get('name')
     state = first_result.get('admin1').get('#text') or ''
     country = first_result.get('country').get('#text') or ''
-    uzip = first_result.get('postal').get('#text') or ''
-    bot.reply('I now have you at WOEID %s (%s%s, %s, %s %s)' %
-              (woeid, neighborhood, city, state, country, uzip))
+    # uzip = first_result.get('postal').get('#text') or ''
```

Comment:
This whole section, from here to the end of the diff, is messy. The commented-out code should simply be removed.